### PR TITLE
MarkUnknown checkbox visible with TranslateDoc - Fix #146

### LINKED ISSUE #146: Mark unknown is sent with document translation
but it isn't visible to the user.

- if APy actually uses it, then it should be made visible to the user.
- if APy does not actually use it, it should not be sent.

### PR DESCRIPTION
**Pull request by share-with-me**
`markUnknown` checkbox is visible to the user wherein the user can tick/untick the checkbox to achieve document translation. A different ID is used for checkbox in `translateDoc` div so that it can be independently mapped to the XHR data.

fixes #146

### CODE REVIEW

**Comment by sushain97** [comment]
Corresponding changes would have to be made to `persistence.js`. Honestly, I think it would be fine for the option to be shared between the text and document translation mode. The only change necessary in that case would be making it visible in document translation. What do you think?

**Comment by share-with-me** [comment]
> The only change necessary in that case would be making it visible in document translation. What do you think?

Yes! This was my initial approach. However, it was getting a bit difficult to hide `translateText` div and then display a `child` of it along with `docTranslation` - the `markunknown checkbox`. This [stackoverflow link](https://stackoverflow.com/questions/5521387/show-child-div-within-hidden-parent-div) discusses that it is possible to display a child modifying `parent` 's visibility but not with `display: none`. 

One solution I thought of is: Instead of 

`$('div#translateText').fadeOut('fast', function () {`, I would use

`$('#originalText, #translatedText, #translateDoc').fadeOut('fast', function () {` ... thus not altering the display CSS of the checkbox. Seems like a good approach?

**Comment by sushain97** [comment]
> $('#originalText, #translatedText, #translateDoc').fadeOut('fast', function () { ... thus not altering the display CSS of the checkbox. Seems like a good approach?

I would add a class to those elements instead and trigger that.

**Comment by share-with-me** [comment]
> I would add a class to those elements instead and trigger that.

I did the same :D . Thanks!

**Comment by share-with-me** [comment]
I make use of `siblings` method. This did help reduce a lot of code. Does this approach make sense?

File: index.html.in [hunk 1]
```diff
@@ -191,9 +191,9 @@
                                           <i class="fa fa-file-text"></i> <span data-text="Translate_Document">Translate a document</span>
                                       </button>
                                   </div>
-                                  <div class="col-lg-3 col-md-6  pull-right">
+                                  <div class="col-lg-3 col-md-6 pull-right" id="translateOptionsContainer">
```
Comment (sushain97) [inline index.html.in]: So, I thought about this a bit and I'm not actually a huge fan of the siblings solution. It's clever and is probably the minimal diff or something but it requires carefully inspecting the HTML to figure out what things are siblings, etc. I would prefer a more explicit approach, e.g.

hiding/showing `#translateText *:not(#translateOptionsContainer), #translateOptions *:not(#markUnknownContainer)`?

Comment (share-with-me) [inline index.html.in]: Sure. I would get on with this approach.

Comment (share-with-me) [inline index.html.in]: Done.

File: assets/js/translator.js [hunk 2]
```diff
@@ -147,6 +147,9 @@ if(modeEnabled('translation')) {
             if($('div#translateText').is(':visible')) {
                 translateText();
             }
+            else if($('div#docTranslation').is(':visible')) {
+                translateDoc();
+            }
```
Comment (share-with-me) [inline assets/js/translator.js]: Um, I am not sure why this is appearing in the diff here :/ . I simply did a `git rebase` with master. Do you suggest making a separate PR? (The below snippet has the changes that you asked to do @sushain97 )!

Comment (sushain97) [inline assets/js/translator.js]: Hm, maybe you're rebasing with your master instead of `origin/master`? I think these changes are separate so we don't want to merge them.

Comment (share-with-me) [inline assets/js/translator.js]: Um, I did a `git fetch origin, git rebase origin/master`. I am not sure why it still exists.

**Comment by share-with-me** [comment]
Closing here and continuing on #180
